R/sass.R

```diff
@@ -35,13 +40,20 @@
 #'   sass_file(tmp_file)
 #' ))
 sass <- function(input = NULL, options = sass_options(), output = NULL,
-  cache_options = sass_cache_options()) {
+  cache_options = sass_cache_options(), write_attachments = FALSE) {
```

Comment:
Maybe this should default to `write_attachments = "auto"` which would resolve to `TRUE` if layers are present

Reply:
Compromise approach of `write_attachments = NULL` would detect the presence of attachments and warn.